pkg/controllers/sync/controller.go

```diff
@@ -571,11 +575,12 @@ func (s *SyncController) syncToClusters(fedResource FederatedResource, collision
 		// Versioning of federated resources is an optimization to
 		// avoid unnecessary updates, and failure to record version
 		// information does not indicate a failure of propagation.
-		runtime.HandleError(err)
+		federatedKeyLogger.WithValues("error-info", err).V(3).Info("Failed to record version information")
```

Comment:
```suggestion
		federatedKeyLogger.Error(err, "Failed to record version information")
```

Reply:
The comment seems to indicate that this err will not cause the propagation to fail. Does it need to be changed to an error here?

Reply:
We should log it as an error for visibility. Logging it does not cause the propagation to fail.